### PR TITLE
Add a histogram that shows the distribution of heights of reply trees

### PR DESCRIPTION
**Pull request by IshaniSanturkar**
In this PR,
- I moved the charts to the comments page 
- I added a chart which displays the distribution of lengths of top-level reply trees currently on the page
image

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ReplyTreeChartServlet.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
```
Comment (EatingW) [inline portfolio/src/main/java/com/google/sps/servlets/ReplyTreeChartServlet.java]: Oops I guess I overlooked it until now, but I think you should change the files you created to Copyright 2020 (although I think it probably doesn't matter)

File: portfolio/src/main/java/com/google/sps/servlets/ReplyTreeChartServlet.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/replytree-chart")
+public class ReplyTreeChartServlet extends HttpServlet {
+
+  /*
+   * Called when a client submits a POST request to the /chart URL
```
Comment (EatingW) [inline portfolio/src/main/java/com/google/sps/servlets/ReplyTreeChartServlet.java]: This comment looks like it's about numcomment-chart :)

File: portfolio/src/main/java/com/google/sps/servlets/NumCommentChartServlet.java [hunk 3]
```diff
@@ -30,8 +30,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/chart")
-public class ChartServlet extends HttpServlet {
+@WebServlet("/numcomment-chart")
+public class NumCommentChartServlet extends HttpServlet {
 
   /*
    * Called when a client submits a POST request to the /chart URL
```
Comment (EatingW) [inline portfolio/src/main/java/com/google/sps/servlets/NumCommentChartServlet.java]: nit: s/chart/numcomment-chart

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -592,16 +592,41 @@ function drawChart() {
...
       };
-      const chart = new google.visualization.LineChart(document.getElementById("chartdiv"));
+      const chart = new google.visualization.LineChart(document.getElementById("numcommentchartdiv"));
+      chart.draw(data, options)
+    });
+  fetch("/replytree-chart")
+    .then(response => response.json())
+    .then(replyTreeLength => {
+      const data = new google.visualization.DataTable();
+      //   data.addColumn("string", "ID");
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js]: nit: delete?

**Review by EatingW — CHANGES_REQUESTED** [review]
Top level comment: Instead of having the bar chart display a label "Reply tree length (count)", maybe it would be clearer to label the x axis and the y axis?

File: portfolio/src/main/java/com/google/sps/servlets/NumCommentChartServlet.java (hunk 3 above)
Comment (IshaniSanturkar) [reply portfolio/src/main/java/com/google/sps/servlets/NumCommentChartServlet.java]: Did you mean replacing /chart with /numcomment-chart in the comment?

Comment (EatingW) [reply portfolio/src/main/java/com/google/sps/servlets/NumCommentChartServlet.java]: yup that's what I mean

**Review by EatingW — APPROVED** [review]

**Pull request merged**